ci/bump_version.sh

```diff
@@ -1,51 +1,63 @@
-#!/bin/bash -xv
+#!/bin/bash
 
 if [ $# != 1 ]
 then
      echo "Please provide version as the first argument"
      exit 1
 fi
 
-version=$1
+echo "Searching for crates..."
+
+crates=($(find . -type d -exec sh -c '[ -f "$0"/Cargo.toml ]' '{}' \; -print))
```

Comment:
Would like to see you try `bash` here instead of `sh` and try to stay in the same command shell.

Reply:
Bash works too.

There is something that I tried on my terminal. If I use `sh` along with an argument, it keeps me inside the `bash` shell. Similarly if I do something similar in sh shell, that is I use `bash` along with an argument it'll also keep me in the same shell.

However, if I use `bash` or `sh` without an arguments, that switches me to a new shell.